server/src/main/resources/templates/web/eastwest/login.html

```diff
@@ -41,6 +41,21 @@ <h4 class="modal-title">This is a U.S. government service.</h4>
 
 
         <div class="js-full_login">
+
+        <div class="alert alert-danger">
+          <div class="panel-body">
+            <h4 class="h4-primary">
+              IMPORTANT: This hosting environment will be shut down this month.
+            </h4>
+            <p>
+              If you haven't already <a href="https://cloud.gov/docs/apps/govcloud/#migration" style="color: white; text-decoration: underline;">migrated your app(s)</a> to the new GovCloud hosting environment, please do so by March 15th.
```

Comment:
Would the 'dark background' styling from https://standards.usa.gov/components/typography/#links work instead of inline styling?